### PR TITLE
Add order field to tasks

### PR DESCRIPTION
**Pull request by GodVenn**

### CODE REVIEW

File: backend/api/Program.cs [hunk 1]
```diff
@@ -16,17 +16,14 @@
 
 builder.AddAzureEnvironmentVariables();
 
-if (!builder.Environment.EnvironmentName.Equals("Test", StringComparison.OrdinalIgnoreCase))
-{
-    // The ExcludeSharedTokenCacheCredential option is a recommended workaround by Azure for dockerization
```
[the remark this answers is not in the document]
Comment (GodVenn) [reply backend/api/Program.cs:21]: You are right, will revert

**Review by Christdej — APPROVED** [review]

**Review by Christdej — APPROVED** [review]
LGTM

**Pull request merged**